provider_shopper/lib/models/catalog.dart

```diff
@@ -41,6 +41,8 @@ class CatalogModel {
     // is also its id.
     return getById(position);
   }
+
+  static List<String> getCatalogItems() => _itemNames;
```

Comment:
Rather than add a property here, you could simply "unhide" _itemNames by removing the underscore. It's a constant, so it can't be modified anyway.